src/systemtraynotificationhandler.h

```diff
@@ -32,13 +33,13 @@ class SystemTrayNotificationHandler : public NotificationHandler {
   virtual void updateIcon();
 
  protected:
-  QMenu m_menu;
-  QSystemTrayIcon m_systemTrayIcon;
+  QMenu* m_menu;
+  QSystemTrayIcon* m_systemTrayIcon;
 
  private:
   void createStatusMenu();
 
-  void maybeActivated(QSystemTrayIcon::ActivationReason reason);
+  void maybeActivated(int reason);
```

Comment:
Let's keep the include `QSystemTrayIcon>` and let's have a reason enum. It seems easier to understand when reading the code.

Reply:
How would you feel about having a comment here about QSystemTrayIcon::ActivationReason? Unless there is a way for me to import only this enum and not the whole QSystemTrayIcon class here